canary/e2e/package.json

```diff
@@ -10,19 +10,20 @@
     "test": "TZ=UTC cypress run --spec 'features/**/*.feature'"
   },
   "cypress-cucumber-preprocessor": {
-    "nonGlobalStepDefinitions": true,
-    "nonGlobalStepBaseDir": "cypress/integration",
-    "stepDefinitions": "features"
+    "filterSpecs": true,
+    "stepDefinitions": [
+      "cypress/integration/common/*.ts"
+    ]
   },
   "author": "",
   "devDependencies": {
-    "cypress": "^9.5.1",
-    "@testing-library/cypress": "^7.0.6",
-    "@types/cypress-cucumber-preprocessor": "^4.0.0",
-    "cypress-cucumber-preprocessor": "^4.1.2",
+    "@badeball/cypress-cucumber-preprocessor": "^16.0.3",
```

Comment:
Replacing with same library used in e2e package, `cypress-cucumber-preprocessor` hasn't been updated in the past 2 years: https://www.npmjs.com/package/cypress-cucumber-preprocessor